tools/go-cli/go-whisk-cli/commands/rule.go

```diff
@@ -404,7 +404,7 @@ var ruleGetCmd = &cobra.Command{
         }
 
         if (flags.rule.summary) {
-            fmt.Fprintf(color.Output, "rule /%s/%s\n", rule.Namespace, rule.Name)
+            fmt.Fprintf(color.Output, "%s /%s/%s\n", boldString("rule"), rule.Namespace, rule.Name)
```

Comment:
@rabbah - Should the `wsk rule get RULE --summary` command output the active/inactive status with the full rule name?  Currently, only the full name is displayed.